### PR TITLE
add ignore files

### PR DESCRIPTION
**Pull request by zaliqarosli**
### Brief summary of changes
add files to git ignore so compiled js files don't show up when trying to change branches

### CODE REVIEW

**Comment by zaliqarosli** [comment]
@ridz1208

**Comment by maltheism** [comment]
Hi @zaliqarosli, I always see the below in my `git status` after running `npm install`:
```
modules/conflict_resolver/js/
modules/data_integrity_flag/js/
modules/dicom_archive/js/
modules/examiner/js/
modules/imaging_browser/js/imagingBrowserIndex.js
modules/instrument_manager/js/
modules/issue_tracker/js/
modules/media/js/
modules/quality_control/js/
modules/server_processes_manager/js/
modules/user_accounts/js/userAccountsIndex.js
```
Is there a way to add these to ignore as well.

**Comment by zaliqarosli** [comment]
@maltheism hey, are they tracked or untracked? the ones I added in this PR are tracked which makes it anyone for when you want to switch branches. but i think they ones you provided above are untracked right? so they should be somewhat "ignored" for branch switches

**Comment by maltheism** [comment]
@zaliqarosli They're untracked but I wish they were not there because I cannot use `git add *` instead of having to manually type everything to git add.

**Comment by zaliqarosli** [comment]
@maltheism i think once everything is reactified, and we remove pure js, we can gitignore `modules/*/js/*.js`. but for now, i think the only way is to add each directory to the main .gitignore file, which might be too specific for a base file? not too sure. i wonder what @ridz1208 and @driusan think

File: modules/conflict_resolver/js/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+resolved_conflicts_columnFormatter.js
+unresolved_columnFormatter.js
```
Comment (driusan) [inline modules/conflict_resolver/js/.gitignore]: Shouldn't all of these be `*.js`?

**Comment by driusan** [comment]
@zaliqarosli There's a few modules which have (tracked, non-jsx) javascript directly in the js folder, so that pattern doesn't work for now.

**Comment by zaliqarosli** [comment]
@driusan i'm confused. so i should revert this back to the first commit?

**Comment by maltheism** [comment]
@zaliqarosli, I think your pattern works for the modules you're using it for but not for the other modules that have js files (that are not compiled from weback into the js directory).

**Comment by driusan** [comment]
no, I think it looks good now.. I was referring to your suggestion of adding a pattern for `modules/*/js/*.js`

**Pull request merged**